test_files/reaction_network_files/H.xyz

```diff
@@ -0,0 +1,3 @@
+1
+
+H          0.15844        0.48081        1.23328
```

Comment:
...why not 0 0 0?

Reply:
I pulled these all directly from the pymatgen test files. Should this be 0 0 0?